test/test_blocks.py

```diff
@@ -1,10 +1,6 @@
 import random
 import unittest
-
-try:
-    from .test_data.blocks_configs import XRAY_CONFIG  # for pytest
-except ImportError:
-    from test_data.blocks_configs import XRAY_CONFIG  # for regular python
+from test.test_data.blocks_configs import XRAY_CONFIG
```

Comment:
Can we revert this change? Without this, I get import errors:

```
ModuleNotFoundError: No module named 'test.test_data'
```

Reply:
Can you try `python -m pytest test/test_blocks.py`? Or run the whole tests with `bash scripts/run_all_tests.sh`.
If possible I would like to stay away from this syntax.